docs/source/guide/tutorials/base/deploy.rst

```diff
@@ -1,19 +1,28 @@
-How to deploy the model
-=======================
+How to deploy the model and use deployed demo
+=============================================
 
-This guide shows, how to deploy a model trained in the :doc:`previous stage <how_to_train/index>`. 
-As a result of this step, we'll get the exported model together with the self-contained python package, a demo application to port and infer it outside of this repository.
+This guide shows, how to deploy a model trained in the :doc:`previous stage <how_to_train/index>` and visualize it outside of this repository.
+As a result of this step, we'll get the exported model together with the self-contained python package and a demo application to visualize results in other environment without long installation process.
+
+.. NOTE::
+    To learn how to use CLI demo inside this tutorial with more features, refer to :doc:`demo`.
```

Comment:
where is the demo file?

Reply:
You review and find my mistakes as fast as I do it, who are you doing this?